webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2460,6 +2463,43 @@ var CURRENT_GROUP_DATA={};
                 mck_text_box.addEventListener("keyup",function(){
                     _this.toggleMediaOptions(this);
 
+                });
+                $mck_text_box.keyup(function (){
+                    if(CHAR_CHECK){
+                        var warning_box = document.getElementsByClassName('mck-char-warning')[0];
+                        var char_count = document.getElementsByClassName('mck-char-count')[0];
+                        var str = ($mck_text_box.text().toString()).trim();
+                        var len = str.length;
+                        if(len > 199){
+                            if(len > 256){
+                                var rem = len - 256; 
+                                var remtxt ="(Remove " + rem + " characters)";
+                            }else{
+                                var rem = 256 - len;
+                                var remtxt = "(" + rem + " characters remaining)";
+                            }
+                            char_count.innerHTML = remtxt;
+                            if(warning_box.classList.contains("n-vis")){
+                                warning_box.classList.remove("n-vis");
+                                if(!(warning_box.classList.contains('warning-slide-up'))){
+                                    warning_box.classList.add('warning-slide-up');
+                                    warning_box.classList.remove('warning-slide-down');
+                                }
+                            }
+                        }else{
```

Comment:
@hardikparashar pls format the code, add spaces

Reply:
done

Reply:
still showing same }else{

Reply:
resolved in new commit, please check